Shared/Samples/Browse organization basemaps/BrowseOrganizationBasemapsView.swift

```diff
@@ -0,0 +1,136 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct BrowseOrganizationBasemapsView: View {
+    @StateObject private var model = Model()
+    
+    var body: some View {
+        Form {
+            switch model.basemaps {
+            case .success(let basemaps):
+                ForEach(basemaps, id: \.item?.id?.rawValue) { basemap in
+                    Button(basemap.title) {
+                        model.selectedItem = .init(basemap: basemap)
+                    }
```

Comment:
```suggestion
                    Button {
                        model.selectedItem = .init(basemap: basemap)
                    } label: {
                        HStack {
                            Text(basemap.title)
                            Spacer()
                        }
                        .contentShape(Rectangle())
                    }
```

This allows a basemap to be selected when tapping anywhere on the list item row rather than just the label.